src/styles/components/containers/_panel.scss

```diff
@@ -7,6 +7,7 @@
   border-right: var(--border-default);
   box-sizing: border-box;
   position: relative;
+  width: calc(var(--panel-width));
```

Comment:
restoring style I deleted earlier which was necessary for the color scale picker